extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -211,7 +211,7 @@ export class AmpIframe extends AMP.BaseElement {
           this.element);
       return;
     }
-    this.requestChangeHeight(newHeight);
+    this./*REVIEW*/changeHeightIfAllowed(newHeight);
```

Comment:
changed it to attemptChangeHeight - let me see if this will skip the presubmit check